shared/api/types.ts

```diff
@@ -81,15 +83,21 @@ export type JsonCategoryType = {
 
 export type JsonChatMessageType = {
   id: number
-  body: string
+  content: string
   user_is_author: boolean
   automatic_answer: boolean
 }
 
 export type JsonChatMessagesType = {
+  chatbot_typing: boolean
   messages: JsonChatMessageType[]
 }
 
+export type ChatMessages = {
```

Comment:
I am not sure whether it should be here...

Reply:
🔧 I would say no as these are only types for the api response. I'd just move it to `mapping/mapChatMessages` (currently `buildChatMessages`).